KingfisherWebP/Classes/CGImage+WebP.m

```diff
@@ -427,9 +431,13 @@ CFDataRef WebPDataCreateWithAnimatedImageInfo(CFDictionaryRef imageInfo) {
         WebPPictureInit(&frame);
         if (WebPPictureImportCGImage(&frame, (CGImageRef)CFArrayGetValueAtIndex(imageFrames, i))) {
             WebPConfig config;
-            WebPConfigInit(&config);
-            WebPConfigLosslessPreset(&config, 0);
-            WebPAnimEncoderAdd(enc, &frame, (int)(frameDurationInMilliSec * i), &config);
+            if (isLossy) {
+                WebPConfigInitInternal(&config, WEBP_PRESET_DEFAULT, quality, WEBP_ENCODER_ABI_VERSION);;
```

Comment:
;;🤣

Reply:
As the name indicates, `WebPConfigInitInternal` is an internal function, I suggest we could use the public counterpart `WebPConfigPreset`

Reply:
Thanks, @Yeatse for the good suggestion

Reply:
Thanks, @Yeatse for the good suggestion.